tests/system/src/test/java/test/com/sun/javafx/text/TextLayoutTest.java

```diff
@@ -0,0 +1,521 @@
+/*
+ * Copyright (c) 2012, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
weird, github shows this file as new, but it is not.
2024

Reply:
It's sort of new, it's moved, it used to reside in the graphics project, but as it needs real fonts to its testing work, it had to be a system test.